.github/dependabot.yml

```
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  - package-ecosystem: pip
```

Since this project uses `poetry`, should this be changed?
```suggestion
  - package-ecosystem: poetry
```

---

No, Dependabot for Poetry uses the `pip` setting: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

---

oh, my bad, I misread that page then.